dsl_parser/elements/policies.py

```diff
@@ -256,7 +267,9 @@ class PolicyInstanceType(Element):
 
     def validate(self):
         scaling_policy = constants.SCALING_POLICY
-        if self.initial_value != scaling_policy:
+
+        # Checking for namespaced option
+        if scaling_policy not in self.initial_value:
```

Comment:
Any chance of problems with namespaces that shadow saved names? E.g. namespace that's called `policies` or `node_types` or `inputs`? Maybe we should not allow any of those at all?

Reply:
I don't think this should cause a problem, because those top lvl elements are out of scope for possible namespaced elements, do u think I should add a unit test about this?